metadata-db-app/src/migrations/052_add_coll_ids_catalog_acls.clj

```diff
@@ -0,0 +1,69 @@
+(ns migrations.052-add-coll-ids-catalog-acls
+  (:require [clojure.java.jdbc :as j]
+            [clojure.edn :as edn]
+            [cmr.common.util :as util]
+            [config.migrate-config :as config]
+            [config.mdb-migrate-helper :as h]
+            [cmr.metadata-db.data.oracle.providers :as providers]
+            [clojure.string :as string]))
+
+(defn- get-concept-ids
+  "For a given provider and entry-titles in collection-identifier, returns list of collection concept ids"
+  [provider-id collection-identifier]
+  (let [entry-titles (get collection-identifier :entry-titles)
+        entry-titles (when (seq entry-titles)
+                       (string/join "," (map #(str "'" % "'") entry-titles)))]
+    (when entry-titles
+      (let [provider (-> (h/get-provider provider-id)
+                         providers/dbresult->provider)
+            t (h/get-provider-collection-tablename provider)]
+        (for [result (h/query (format "select concept_id from metadata_db.%s where
```

Comment:
It would be better to parameterize the entry-titles bit of this (you can't parameterize the table name) just to make sure the entry-titles get sanitized before executing the SQL.